File: Practice/src/introduction_coding_tests/AgeOfAlienPlanet.js
function solution(age) {
    var answer = '';
    for (let i = age; i > 0; i = parseInt(i / 10)) {
        answer = String.fromCharCode((i % 10) + 97) + answer;
    }
    return answer;
    // return age
    //     .toString()
    //     .split('')
    //     .map((v) => 'abcdefghij'[v])
    //     .join('');
}

File: Practice/src/introduction_coding_tests/AgeOfAlienPlanet.java
package introduction_coding_tests;

public class AgeOfAlienPlanet {
//	외계행성의 나이
	public String solution(int age) {
		String answer = "";
		for (int i = age; i > 0; i /= 10) {
			answer = (char) (i % 10 + 'a') + answer;
		}
		return answer;
	}
}